daffodil-core/src/main/scala/org/apache/daffodil/dsom/IIBase.scala

```diff
@@ -165,7 +165,13 @@ abstract class IIBase( final override val xml: Node, xsdArg: XMLSchemaDocument,
     res
   }.value
 
-  final lazy val schemaLocationProperty = getAttributeOption("schemaLocation")
+  final lazy val schemaLocationProperty = {
+    val prop = getAttributeOption("schemaLocation")
+    if (prop.isDefined && prop.get == "edu/illinois/ncsa/daffodil/xsd/built-in-formats.xsd") {
+      SDW("schemaLocation property uses deprecated include/import of edu/illinois/ncsa/daffodil/xsd/built-in-formats.xsd. Use org/apache/daffodil/xsd/DFDLGeneralFormatDefinition.dfdl.xsd instead and change the dfdl:format ref to \"GeneralFormat\".")
```

Comment:
So will all our tests that use daffodilTest1 or daffodilNewProperties1 (which is also defined in built-in-formats) all produce SDWs now, or is there a way they are suppressed for our own tests? daffodilNewProperties1 is used quite a bit.

Reply:
All the tests in the Daffodil repo have been converted to using DFDLGeneralFormat now. So only external schemas will get warnings, which I'd rather not have a way to suppress. We want this annoying so people will fix it and we can remove it.

There were only a handful of tests that actually needed daffodilNewProperties1 so I just got rid of the use that format and defined all the necessary properties in individual tests. When we add new properites, these tests will likely break, but it's pretty it's easy to just add the new properties to them, there's only like 5 or so, and we won't be adding new properties all that often.